db/job_context.h

```diff
@@ -90,10 +90,12 @@ struct JobContext {
   struct CandidateFileInfo {
     std::string file_name;
     uint32_t path_id;
```

Comment:
One more thing.. is `path_id` still needed now that we know the full path with `file_path` + `file_name`?

Reply:
The current implementation of SSTFileManager supports only deletes from path_id=0. In DeleteSSTFile, path_id is needed to decide whether file is deleted using sstfilemanager or by call to env->DeleteFile().

Reply:
Oh right that's a good point. I actually wanted to get rid of that restriction as the assumption for which it was introduced (trash files were kept in a separate directory) is no longer true. Let me see if it'd be difficult.

Reply:
#3544